config_script/process_config.py

```diff
@@ -252,9 +252,27 @@ def add_fullstory_config(self, config, plist):
         fullstory = config.get('FULLSTORY', {})
         enabled = fullstory.get('ENABLED')
         orgID = fullstory.get('ORG_ID')
+        recordOnStart = fullstory.get('RECORD_ON_START')
+        swiftUIEnabled = fullstory.get('SWIFTUI_ENABLED')
+        swiftUISelectorVersion = fullstory.get('SWIFTUI_SELECTOR_VERSION')
+        swiftUIPreviewVersion = fullstory.get('SWIFTUI_PREVIEW_VERSION')
 
         if enabled and orgID:
-            plist["FullStory"] = {"orgID": orgID}
+            plist["FullStory"] = {
+            "orgID": orgID,
+            "RecordOnStart": recordOnStart,
+            "SwiftUIEnabled": swiftUIEnabled,
+            "SwiftUISelectorVersion": swiftUISelectorVersion,
+            "SwiftUISelectorPreview": swiftUIPreviewVersion
+            }
+        else:
+            plist["FullStory"] = {
+            "orgID": 'o-1XJF5N-na1',
```

Comment:
Please don't hardcode this key. It's better to move this to the dev and stage environment. We will see what to do when the keys will be missing in the config file (openedX)